server/src/main/java/org/elasticsearch/search/suggest/SuggestionBuilder.java

```diff
@@ -306,7 +306,7 @@ protected void populateCommonFields(QueryShardContext context, SuggestionSearchC
 
         MappedFieldType fieldType = context.getFieldType(field);
         if (analyzer == null) {
-            suggestionContext.setAnalyzer(context.getSearchAnalyzer(fieldType));
+            suggestionContext.setAnalyzer(fieldType.getTextSearchInfo().getSearchAnalyzer());
```

Comment:
here we don't need a delegating analyzer because we don't do cross fields, but rather the field name that the analyzer will get as an argument is always the same field name which we the suggestions are pulled from?

Reply:
Correct.